massband/diffusion/arrhenius.py

```
@@ -0,0 +1,227 @@
+            ax.scatter(
+                1000 / ref_data["temperatures"],
+                ref_data["diffusion"],
+                marker="x",
+                color="orangered",
+                zorder=9,
+                label="Ref. Data",
+                s=50,
+            )
+
+        # Plot credible intervals
+        for i, (ci, sigma) in enumerate(zip(credible_intervals, sigmas)):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)